contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -559,6 +559,47 @@ namespace eosiosystem {
          [[eosio::action]]
          void bidrefund( name bidder, name newname );
 
+         using init_action = eosio::action_wrapper<"init"_n, &system_contract::init>;
+         using onblock_action = eosio::action_wrapper<"onblock"_n, &system_contract::onblock>;
```

Comment:
No need for `onblock`. No one should be trying to send an inline `eosio::onblock` action.